test/unit/spec/common/simpleLogViewer-spec.js

```diff
@@ -359,9 +359,10 @@ define([
                             this.simpleLogViewerInstance.messages.BATCH_JOB
                         );
                         expect(this.bus.emit.calls.allArgs()).toEqual(
-                            jasmine.arrayWithExactContents([
-                                [jcm.MESSAGE_TYPE.STATUS, { [jcm.PARAM.JOB_ID]: jobId }],
```

Comment:
no longer send a job status request at start up -- that's handled by the job manager